### ISSUE TITLE
www.slideshare.net breaks element

### ISSUE THREAD

**Issue by terrorist96** [EFForg/privacybadger#1100]
~~http://www.slideshare.net/~~
~~http://www.slideshare.net/geniusworks/xmarketing-the-new-rules-of-marketing~~

~~`cdn.slidesharecdn.com` and `image.slidesharecdn.com` break images~~

**Comment by terrorist96** [comment]
https://blog.iota.org/a-primer-on-iota-with-presentation-e0a6eb2cc621

`www.slideshare.net` breaks embed.

**Comment by ghostwords** [comment]
`slidesharecdn.com` should be fixed as of Privacy Badger version 2017.6.13.1 (by #1403 and #1428). Might still run into issues with `www.slideshare.net`, need to check.

**Comment by terrorist96** [comment]
@ghostwords Yes. Issues in first post are fixed, but issue in second post is still outstanding.

**Comment by terrorist96** [comment]
https://www.endothelix.com/

`Why Should We Measure Endothelial Function?` section
`www.slideshare.net`
